Makefile.am

```diff
@@ -343,6 +343,14 @@ clean-local:
 	rm -f $(git_include) $(git_include).in
 endif
 
+.PHONY: my-prof.supp
+my-prof.supp:
+	@sed '/^# AUTO-GENERATED START/q' prof.supp > $@
+	@cat /usr/share/glib-2.0/valgrind/glib.supp >> $@
+	@wget -O- https://raw.githubusercontent.com/python/cpython/refs/tags/v`python3 --version | cut -d' ' -f2`/Misc/valgrind-python.supp >> $@
+	@wget -O- https://raw.githubusercontent.com/GNOME/gtk/refs/tags/`pkg-config --modversion gtk+-3.0`/gtk.supp >> $@
```

Comment:
This one isn't working: `https://raw.githubusercontent.com/GNOME/gtk/refs/tags/gtk.supp`
This one is though: `https://raw.githubusercontent.com/GNOME/gtk/refs/heads/main/gtk.supp`

Reply:
> This one is though: `https://raw.githubusercontent.com/GNOME/gtk/refs/heads/main/gtk.supp`

You most likely don't want the supp file for main, but for "your installed gtk version".

> This one isn't working: `https://raw.githubusercontent.com/GNOME/gtk/refs/tags/gtk.supp`

This is wrongly extended from 
```
https://raw.githubusercontent.com/GNOME/gtk/refs/tags/`pkg-config --modversion gtk+-3.0`/gtk.supp
```
, so it seems like `pkg-config --modversion gtk+-3.0` does not work for you.

Do you know where this fails for you?

What do the following commands return?

```
pkg-config --modversion gtk+-3.0
pkg-config --modversion gtk+-2.0
```

Reply:
Ah indeed. `gtk3-devel` wasn't installed. With that it works fine.